### PR TITLE
Implemented CameraEffects override exposure

### LINKED ISSUE #53521: Vulkan: CameraEffects Override Exposure not implemented
### Godot version

4.0.dev (2fc31fdfc)

### System information

Fedora 34, GeForce GTX 1080 (NVIDIA 470.57.02)

### Issue description

CameraEffects' Override Exposure property does nothing, even if Auto Exposure is enabled in WorldEnvironment.

A code search reveals that the exposure override isn't used in any shader or in the autoexposure code: https://github.com/godotengine/godot/search?q=override_exposure

### Steps to reproduce

- Add a WorldEnvironment node. Add an Environment resource to it with **Auto Exposure** enabled.
- Add Camera3D node.
- Add a CameraEffects resource to the Camera3D's **Effects** property.
- Enable **Override Exposure** in CameraEffects. Set to a very low or very high exposure value to make the difference with auto exposure obvious.
- Enable camera preview in the editor or run the project. Notice how exposure isn't overridden (auto exposure will still be in action).

### Minimal reproduction project

[test_camera_override_exposure.zip](https://github.com/godotengine/godot/files/7303437/test_camera_override_exposure.zip)

### PR DESCRIPTION
**Pull request by ator-dev**
<i>Bugsquad edit</i>: Fix #53521

### CODE REVIEW

File: servers/rendering/renderer_rd/renderer_scene_render_rd.cpp [hunk 1]
```diff
@@ -2086,7 +2086,7 @@ void RendererSceneRenderRD::_render_buffers_post_process_and_tonemap(const Rende
 	ERR_FAIL_COND(!rb);
 
 	RendererSceneEnvironmentRD *env = environment_owner.get_or_null(p_render_data->environment);
-	//glow (if enabled)
+	// glow and override exposure (if enabled)
```
Comment (LightningAA) [inline servers/rendering/renderer_rd/renderer_scene_render_rd.cpp:2089]: While you're at it 😄
```suggestion
	// Glow and override exposure (if enabled).
```
https://docs.godotengine.org/en/stable/community/contributing/code_style_guidelines.html#comment-style-guide

**Review by AaronRecord — CHANGES_REQUESTED** [review]

File: servers/rendering/renderer_rd/renderer_scene_render_rd.cpp [hunk 2]
```diff
@@ -2280,6 +2284,8 @@ void RendererSceneRenderRD::_post_process_subpass(RID p_source_texture, RID p_fr
 	ERR_FAIL_COND(!rb);
 
 	RendererSceneEnvironmentRD *env = environment_owner.get_or_null(p_render_data->environment);
+	// override exposure (if enabled)
```
Comment (ator-dev) [inline servers/rendering/renderer_rd/renderer_scene_render_rd.cpp:2287]:
```suggestion
	// Override exposure (if enabled).
```

**Comment by LightningAA** [comment]
You'll also want to squash the commits, see https://docs.godotengine.org/en/stable/community/contributing/pr_workflow.html#the-interactive-rebase

**Comment by ator-dev** [comment]
Sorry, I somehow messed up my repo completely and had to start over; now I can't seem to reopen the pull request with my newly fixed branch. I've started a new pull request here: [https://github.com/godotengine/godot/pull/54782](url)

**Comment by ator-dev** [comment]
I managed to break it again but I understand what I was doing wrong now. This one should be final: https://github.com/godotengine/godot/pull/54783
(Really sorry, again)

**Comment by LightningAA** [comment]
> I managed to break it again but I understand what I was doing wrong now. This one should be final: #54783 (Really sorry, again)

It's okay, you're learning 😃

**Comment by ator-dev** [comment]
> > I managed to break it again but I understand what I was doing wrong now. This one should be final: #54783 (Really sorry, again)
> 
> It's okay, you're learning smiley
Thank you (and Calinou) for being so patient with me :D
I think all beginners probably lose half a day to Git when they're starting out with it; it's a pretty horrible tool from a user experience point of view. Don't worry, I have now learnt (at great pains and over many hours) how to pretty much use it properly...

**Pull request closed without merging**